distribution/docs/src/main/resources/_contents/_spatial-contents/integrating-spatial-contents.adoc

```diff
@@ -1379,6 +1379,483 @@ xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+                   <ogc:Literal>2012-12-25T10:10:19.000-07:00</ogc:Literal>
+               </ogc:PropertyIsGreaterThan>
+           </ogc:Filter>
+       </csw:Constraint>
+   </csw:Query>
+</csw:GetRecords>
+----
+A query for records created between two specific dates.
+[source,xml,linenums]
+----
+<csw:GetRecords xmlns:ows="http://www.opengis.net/ows" xmlns:ns2="http://www.w3.org/1999/xlink"
+               xmlns:ogc="http://www.opengis.net/ogc" xmlns:gml="http://www.opengis.net/gml"
+               xmlns:csw="http://www.opengis.net/cat/csw/2.0.2"
+               xmlns:ns6="http://www.w3.org/2001/SMIL20/"
+               xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:dct="http://purl.org/dc/terms/"
+               xmlns:ns9="http://www.w3.org/2001/SMIL20/Language" resultType="results"
+               outputFormat="application/xml" outputSchema="http://www.opengis.net/cat/csw/2.0.2"
+               startPosition="1" maxRecords="20" service="CSW" version="2.0.2">
+   <csw:Query typeNames="csw:Record">
+       <csw:ElementSetName>full</csw:ElementSetName>
+       <csw:Constraint version="1.1.0">
+           <ogc:Filter>
+               <ogc:PropertyIsBetween>
+                   <ogc:PropertyName>created</ogc:PropertyName>
+                   <ogc:LowerBoundary>
+                       <ogc:Literal>2012-12-24T17:05:19.000-07:00</ogc:Literal>
+                   </ogc:LowerBoundary>
+                   <ogc:UpperBoundary>
+                       <ogc:Literal>2012-12-26T17:08:52.000-07:00</ogc:Literal>
+                   </ogc:UpperBoundary>
+               </ogc:PropertyIsBetween>
+           </ogc:Filter>
+       </csw:Constraint>
+   </csw:Query>
+</csw:GetRecords>
+----
+A query for records created before a specific date.
+[source,xml,linenums]
+----
+<csw:GetRecords xmlns:ows="http://www.opengis.net/ows" xmlns:ns2="http://www.w3.org/1999/xlink"
+               xmlns:ogc="http://www.opengis.net/ogc" xmlns:gml="http://www.opengis.net/gml"
+               xmlns:csw="http://www.opengis.net/cat/csw/2.0.2"
+               xmlns:ns6="http://www.w3.org/2001/SMIL20/"
+               xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:dct="http://purl.org/dc/terms/"
+               xmlns:ns9="http://www.w3.org/2001/SMIL20/Language" resultType="results"
+               outputFormat="application/xml" outputSchema="http://www.opengis.net/cat/csw/2.0.2"
+               startPosition="1" maxRecords="20" service="CSW" version="2.0.2">
+   <csw:Query typeNames="csw:Record">
+       <csw:ElementSetName>full</csw:ElementSetName>
+       <csw:Constraint version="1.1.0">
+           <ogc:Filter>
+               <ogc:PropertyIsLessThan>
+                   <ogc:PropertyName>created</ogc:PropertyName>
+                   <ogc:Literal>2012-12-26T10:10:19.000-07:00</ogc:Literal>
+               </ogc:PropertyIsLessThan>
+           </ogc:Filter>
+       </csw:Constraint>
+   </csw:Query>
+</csw:GetRecords>
+----
+===== `Spatial Operation Examples`
```

Comment:
Once you get to these spatial examples, you need to provide more description of what each of them does.